conf/machine/imx8mp-evk.conf

```diff
@@ -4,7 +4,7 @@
 #@DESCRIPTION: Machine configuration for NXP i.MX 8M Plus Evaluation Kit
 #@MAINTAINER: Jun Zhu <junzhu@nxp.com>
 
-MACHINEOVERRIDES =. "mx8:mx8m:mx8mp:"
+MACHINEOVERRIDES =. "boot-container:mx8:mx8m:mx8mp:"
```

Comment:
You unconditionally use now U-Boot to provide the boot container if I understand this line correctly.
I'd propose to rename this commit as this commit does actually use the support the other commits are providing. E.g.

conf: machine: mx8m[m,n,p]: use boot container class

Reply:
Yes, for Community BSP - it is added unconditionally. NXP BSP is unaffected here, as it would use `u-boot-imx` instead of -u-boot-fslc`.

Would clarify the commit message in _V2_